.github/workflows/preview.yml

```diff
@@ -15,7 +15,7 @@ jobs:
         with:
           repository: quarkusio/quarkusio.github.io
       - name: Download PR Artifact
-        uses: dawidd6/action-download-artifact@v2
+        uses: actions/download-artifact@v3
```

Comment:
Are you sure that:

> Let's suppose you have a workflow with a job in it that at the end uploads an artifact using actions/upload-artifact action and you want to download this artifact in another workflow that is run after the first one. Official actions/download-artifact does not allow this.

is now possible with `v3`? I don't see it in the changelog.

Reply:
I have used this-https://github.com/marketplace/actions/download-a-build-artifact

Reply:
@gsmet, let  CI trigered

Reply:
I don't have the time to debug this if it doesn't work. And I don't see how this would even work given you haven't changed the inputs and some of them are not supported by the action you're trying to push.

This workflow works and is OK as is.